grid.c

```diff
@@ -790,6 +790,7 @@ grid_string_cells_code(const struct grid_cell *lastgc,
 		{ GRID_ATTR_UNDERSCORE_3, 43 },
 		{ GRID_ATTR_UNDERSCORE_4, 44 },
 		{ GRID_ATTR_UNDERSCORE_5, 45 },
+		{ GRID_ATTR_OVERLINE, 53 },
```

Comment:
The 53 here also looks mostly useless because I'm using a literal in tty.c:2203. I'm trying to figure out where this gets connected to the TTYC_ codes that are used by tty_putcode